std/string.d

```diff
@@ -2184,15 +2184,15 @@ auto representation(Char)(Char[] s) @safe pure nothrow @nogc
  * See_Also:
  *      $(XREF uni, toCapitalized) for a lazy range version that doesn't allocate memory
  */
-S capitalize(S)(S s) @trusted pure
+S capitalize(S)(S input) @trusted pure
```

Comment:
It's really annoying that this module uses `input`, `s`, and `str` seemingly at random, but this isn't a problem with your pull request.